packages/strapi-plugin-upload/admin/src/utils/tests/formatBytes.test.js

```diff
@@ -7,27 +7,27 @@ describe('UPLOAD | components | EditForm | utils', () => {
     });
 
     it('should return 0B if less than 1 bytes is passed', () => {
-      expect(formatBytes(0.9)).toEqual('0B');
+      expect(formatBytes(0.9)).toEqual('900B');
     });
 
     it('should return 1KB if 1024 Bytes is passed', () => {
-      expect(formatBytes(1024)).toEqual('1KB');
+      expect(formatBytes(1024)).toEqual('1000KB');
```

Comment:
It seems that those tests are wrong. 1024B should equal 1KB. Why have they been updated?

Reply:
Because the backend sends the dimensions in kilo bits